src/ParcelRegistry.Api.BackOffice.Handlers.Lambda/Handlers/AttachAddressHandler.cs

```diff
@@ -64,6 +64,7 @@ protected override async Task<ETagResponse> InnerHandle(AttachAddressLambdaReque
         {
             return exception switch
             {
+                ParcelIsRemovedException => ValidationErrors.Common.ParcelRemoved.ToTicketError,
```

Comment:
Can ParcelIsRemovedException be moved to the base ParcelLambdaHandler?

Reply:
It is marked as resolved, but the exception is still here. Did you have any discussion? If so please add it here :)

Reply:
I thought this comment was about the detach handler where I had removed it, will do here as well.